### ISSUE TITLE
Select language for each status page

### ISSUE THREAD

**Issue by barart** [louislam/uptime-kuma#2196]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

The Status Pages are the first face of uptime kuma and you can have many status pages as you want, however the language of the status pages are consistent with the selected language on the dashboard, it would be very useful to specify the language of each status page so you can serve status pages in different languages than the one selected in the dashboard

### ✔️ Solution

On the status page creation/edition add a button like the "show powered by" button but that says... "Main Language" On/Off; when on the language will be the same of the selected in dashboard (so if you change the language on the dashboard it change on the status page as well), when off a user can select the language for that specific status page

**Comment by CommanderStorm** [comment]
it is now possible

**Comment by CommanderStorm** [comment]
actually, I closed the wrong issue.
I meant to close the following very close issue instead:
- https://github.com/louislam/uptime-kuma/issues/1643

**Comment by hahagu** [comment]
So this is not yet resolved?

**Comment by CommanderStorm** [comment]
No, is easy enough to add. Just needs an engineer to look into this for how our plumbing works and how to fix it.

**Comment by hahagu** [comment]
@CommanderStorm 
I've created a draft PR: https://github.com/louislam/uptime-kuma/pull/7437
to resolve this issue.

I've self-tested it, and it seems to work fine.
However, it would be good to have another set of eyes to see the implementation to fix anything if broken.
If not a hassle, would you check out my PR?
